_rules/video-only-media-alternative-fd26cf.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: fd26cf
-name: Video-only as a media alternative for text
+name: Video only as a media alternative for text
```

Comment:
This change is incorrect. This isn't a "video, only as a media alternative". It's a "video-only solution, as a media alternative for text". Please revert.